acceptance/apps/rails_test.go

```diff
@@ -89,7 +89,11 @@ var _ = Describe("RubyOnRails", func() {
 		out, err := env.Epinio("", "apps", "create", rails.Name)
 		Expect(err).ToNot(HaveOccurred(), out)
 
-		// Set the RAILS_MASTER_KEY env variable
+		// Force node version used by buildpack to 16.x
+		out, err = env.Epinio("", "apps", "env", "set", rails.Name, "BP_NODE_VERSION", "16.*")
+		Expect(err).ToNot(HaveOccurred(), out)
+
+		// Provide the expected RAILS_MASTER_KEY to buildpack and application
 		out, err = env.Epinio("", "apps", "env", "set", rails.Name, "RAILS_MASTER_KEY", rails.MasterKey)
 		Expect(err).ToNot(HaveOccurred(), out)
```

Comment:
I guess we can merge these two, but please check out the suggestion because I'm not sure that the indentation or anything is fine! :smile: 

```suggestion
		// Force node version used by buildpack to 16.x
		// and provide the expected RAILS_MASTER_KEY to buildpack and application
		out, err = env.Epinio("", "apps", "env", "set", rails.Name,
			"BP_NODE_VERSION", "16.*",
			"RAILS_MASTER_KEY", rails.MasterKey,
		)
		Expect(err).ToNot(HaveOccurred(), out)
```

Reply:
I have to reject this. Per the syntax help for `epinio app env set ...` the command does not allow assignment of multiple variables and values. (Counter to `push` which allows multiple `--env` options). Checking actual behaviour I confirm that the help is correct.

Reply:
Aaaah, I see, thanks. :+1: